src/ec/packages/ec-component-ecl-compliance/components/ecl-compliance-link.html.twig

```diff
@@ -0,0 +1,23 @@
+{#
+  Parameters:
+  - "link" (object)
+  - "icon" (object)
+#}
+
+{% extends "@ecl-twig/ec-component-ecl-compliance/ecl-compliance.html.twig" %}
+
+{% set not_compliant =
+  (link.label is not defined or link.label is empty) and (
+    (link.icon_position is not defined or link.icon_position is empty) or
+    (icon[0].name is not defined or icon[0].name is empty) or
+    (icon[0].type is not defined or icon[0].type is empty) or
+    (icon[0].path is not defined or icon[0].path is empty)
+  )
+%}
+{% set component = 'link' %}
```

Comment:
related to previous comment if we opt for only a green border, then we don't need to specify these variables..

Reply:
The reason why i added a "success" message is to help identifying the component and the variant that have been tested, the green border we had before was not ok anymore once i introduced the "nested" checks, too confusing.